docs/source/installation_instructions.rst

```diff
@@ -8,8 +8,8 @@ Installation Instructions
    Instructions are listed below for installing Miniconda, but the full user documentation for Anaconda is available at
    https://conda.io/projects/conda/en/latest/user-guide/install/index.html
 
-System Requirments
-------------------
+System Requirements
+-------------------
 
 Before installing Anaconda, the following system requirements must be met:
```

Comment:
they won't be installing Anaconda.  Just miniconda.

Reply:
Fixed, although I am not even sure this statement is true. I may remove it for now.